pkg/querier/worker/worker_test.go

```diff
@@ -232,25 +232,25 @@ func TestQuerierWorker_getDesiredConcurrency(t *testing.T) {
 			},
 			maxConcurrent: 0,
 			expected: map[string]int{
-				"1.1.1.1": 1,
-				"2.2.2.2": 1,
-				"3.3.3.3": 1,
-				"4.4.4.4": 1,
+				"1.1.1.1": MinConcurrencyPerRequestQueue,
+				"2.2.2.2": MinConcurrencyPerRequestQueue,
+				"3.3.3.3": MinConcurrencyPerRequestQueue,
+				"4.4.4.4": MinConcurrencyPerRequestQueue,
 			},
 		},
-		"should create 1 connection for each instance if max concurrency is > 0 but less than the number of in-use instances": {
+		"should create the minimum connections for each instance if max concurrency is less than the minimum min times the number of in-use instances": {
```

Comment:
> the minimum min times the number of in-use instances

I don't think it's fully clear what this means:
* minimum min?
* minimum what?

Reply:
Updated to `"should create the minimum connections for each instance if max concurrency is less than (instances * minimum connections per instance)"`